CONTRIBUTING.md

```diff
@@ -81,6 +81,31 @@ npx ~/repos/provectus/awos/index.js --force-overwrite
 3. ✅ Ensure all files are committed
 4. ✅ Write a clear PR description
 
+## Release Process
+
+### Automated Release Drafting
+
+We use [release-drafter](https://github.com/release-drafter/release-drafter) to automatically compile pull requests into release notes.
+
+### Version Increment Guidelines
+
+PR labels determine version increment:
+
+- `major` label: Increments major version
+- `minor` label: Increments minor version
+- `patch` label: Increments patch version
```

Comment:
Makes sense to also mention that patch version increments automatically even if there are no labels present anywhere